bindings/java/src/main/java/org/ucx/jucx/util/JucxLogger.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+package org.ucx.jucx.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JucxLogger {
```

Comment:
again - do we really need a logger? this is supposed to be a thin wrapper on top of UCP...

Reply:
Are you wanting to throw exceptions instead of logging the errors? In general, printing errors to stdout/stderr creates more problems than using a logger, since whatever Java stack that uses this is probably going to be deep and using loggers for lots of things already.

Reply:
In the JUCX API I throw exceptions in case of errors, in LoadLibrary.java I log the errors because it's executed when Bridge class is loaded to the JVM, "no one is there" to catch the exceptions.